yt_dlp/extractor/rudovideo.py

```diff
@@ -0,0 +1,116 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError, traverse_obj, js_to_json, update_url_query
+
+
+class RudoVideoLiveIE(InfoExtractor):
```

Comment:
With how different the structure of the live and the VOD/Podcast site is, my idea was to have two extractors `RudoVideoLiveIE` for the live content and `RudoVideoIE` for VOD/Podcast content.

Reply:
Imo, the splitting isn't warranted. See ab125964cea3b25ae8f15736ee3c2e308528baa4 (untested)

Reply:
Taking another look at the VOD page, there are in fact more similarities to the live page than I thought.
I initially must have only looked at the pages for podcasts, which are quite different and could warrant splitting. However, given that VODs and podcasts can't be distinguished by the URL, this should indeed be a single extractor.
Sorry for the unnecessary work :pensive:

Reply:
ok, don't worrie.
@seproDev should I name it `RudoVideoLiveIE` or `RudoVideoIE`?

Reply:
oh, now I see @pukkandan's commit. Should I use that commit?

Reply:
I likely has some bugs since I never actually ran the code. But yes, use it as a baseline